ovirtlib4/vms.py

```diff
@@ -97,6 +97,21 @@ def stop_and_wait(self, state='down', *args, **kwargs):
         vm = self.get(wait_for=f"entity.status.value == '{state}'", *args, **kwargs)
         return vm[0] if vm else None
 
+    def get_management_nic(self, mng_network=defaults.MNG_NETWORK):
```

Comment:
I don't think 'mng_network' parameter should be used here. by using it you provide the illusion that one can pass any network, while that's not the case.
Since there can only be one management network in the cluster then this method should figure it out by itself.
```suggestion
    def get_management_nic(self):
```

Reply:
Yap you definitely right, I need to change it all, this is not a good implementation